Formula/tty-share.rb

```diff
@@ -0,0 +1,23 @@
+class TtyShare < Formula
+  desc "Terminal sharing over the Internet"
+  homepage "https://tty-share.com/"
+  url "https://github.com/elisescu/tty-share/archive/v0.6.0.tar.gz"
+  sha256 "4de1a017085cc8cd3b81c696ea33507f507e343bda1ddc26e74d5f6c2d865c70"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
@bayandin we should be getting rid of these right?

Reply:
That's true

@elisescu we're working on removing `GOPATH` from all the formulae (https://github.com/Homebrew/homebrew-core/issues/47627) so it would be nice not to add new formula with it. Also, it could simplify the build process (you can use as examples already converted formulae).

Reply:
Got rid of GOPATH and bumped the upstream version. The new version checks now if stdin is a terminal or not.